debian/vala-panel-appmenu/debian/control

```diff
@@ -13,9 +13,9 @@ Build-Depends: budgie-core-dev,
                libwnck-3-dev (>= 3.4.0),
                libxfce4panel-2.0-dev,
                libxfconf-0-dev,
+               vala-panel-dev (>= 0.3.63),
```

Comment:
0.3.60 reqired here, not 0.3.63 (if for 0.4 series)
for 0.3 series it is 0.3.0<X<0.3.50, where X is vala-panel version